netmanager/src/views/pages/Map/OverlayMap.js

```diff
@@ -344,7 +344,7 @@ export const OverlayMap = ({ center, zoom, heatMapData, monitoringSiteData }) =>
   const mapContainerRef = useRef(null);
   const [map, setMap] = useState();
   const [showSensors, setShowSensors] = useState(true);
-  const [showHeatMap, setShowHeatMap] = useState(false);
+  const [showHeatMap, setShowHeatMap] = useState(true);
```

Comment:
Thanks @FaithDaka , this is great! This aligns with expectations!

1. Could you share some background context/information on why it was `false `previously?
2. Do we have any tradeoffs with this hotfix? If yes, could you share more notes on these tradeoffs?
3. Please check out the attached screenshot with supplementary feedback.

Thanks a lot!

<img width="875" alt="Screenshot 2022-12-08 at 14 22 21" src="https://user-images.githubusercontent.com/1590213/206434714-49bdc0eb-c004-42af-9e99-13b54c3ef9fa.png">

Reply:
@Baalmart 
1. It was _false_ because it was a user preference to see the heatmap not a default map setting.
2. It's not a hotfix. The request was to have the heatmap appear by default load of the public map and that's the implementation.
3. The heatmap still renders objects using client-side resources (the _/predict_ API endpoint hasn't been refactored yet) so that affects it's performance. That's why the previous resolution was to not have it appear by default in order to allow for the calculations to be done based on client-side resources.

Reply:
Thanks @FaithDaka for the clarifications, much appreciated.

About point (3), I guess we need to first improve the performance of the predict service before we proceed with introducing these changes. We do not want to introduce changes with performance issues coupled with them.

Thanks again! I am happy to learn about your additional thoughts on this.

cc: @Codebmk